examples/material_gallery/lib/gallery/home.dart

```diff
@@ -104,18 +104,18 @@ class GalleryHomeState extends State<GalleryHome> {
         children: <Widget>[
           new TwoLevelSublist(
             leading: new Icon(icon: Icons.star),
-            title: new Text("Demos"),
+            title: new Text('Demos'),
             children: <Widget>[
-              new GalleryItem(title: "Weather", builder: () => new WeatherDemo()),
-              new GalleryItem(title: "Fitness", builder: () => new FitnessDemo()),
-              new GalleryItem(title: "Fancy lines", builder: () => new DrawingDemo()),
+              new GalleryItem(title: 'Weather', builder: () => new WeatherDemo()),
+              new GalleryItem(title: 'Fitness', builder: () => new FitnessDemo()),
+              new GalleryItem(title: 'Fancy lines', builder: () => new DrawingDemo()),
               new GalleryItem(title: 'Flexible space toolbar', builder: () => new FlexibleSpaceDemo()),
               new GalleryItem(title: 'Floating action button', builder: () => new TabsFabDemo()),
```

Comment:
How are we determining "best demos"? I mean, the flexible space toolbar is pretty sweet, if you're trying to see how easy it is to make a material app. Some of the "Components" demos are quite compelling too. The "Fitness" demo is great, but if what you want is to see if you can implement some of the more elaborate things in the material design spec, it's not obvious what it's showing you -- you can, after all, get the same effect on a Web page with an animated GIF. :-)

Maybe we should randomise the order and use metrics to determine which demos people look at the most...